### PR TITLE
combined view and pay bills

### PR DESCRIPTION
**Pull request by John-Merritt**

### CODE REVIEW

File: HotelBooking/Views/pay_bill_view.py [hunk 1]
```diff
@@ -101,21 +107,17 @@ def pay_bill(self):
                 bill_id = answer_list[1].strip()
                 self.bill_controller.pay_bill(bill_id)
                 print("\nSuccess!\n")
-        self.show()
+                self.show_again()
+            
+        self.prev_view()
+        
 
     def list_bill(self):
```
Comment (mclhtay) [inline HotelBooking/Views/pay_bill_view.py]: should be `list_bills` since you're printing all bills.

Comment (John-Merritt) [reply HotelBooking/Views/pay_bill_view.py]: petty af

Comment (John-Merritt) [reply HotelBooking/Views/pay_bill_view.py]: also Vincent named it that but Ill change it lol

File: HotelBooking/Views/pay_bill_view.py [hunk 2]
```diff
@@ -6,7 +6,6 @@
 PROMPT_KEY = {
     "OPERATIONS": 'operations',
     "PAY_BILL": 'pay_bill',
-    "LIST_BILL": "list_bill",
```
Comment (mclhtay) [inline HotelBooking/Views/pay_bill_view.py:9]: Why delete this?

Comment (John-Merritt) [reply HotelBooking/Views/pay_bill_view.py:9]: its no longer an option they can pick its just a function that gets called every time theyr on the pay bill page

**Review by mclhtay — APPROVED** [review]
Approve on the basis that the author did manual UI testing.

**Pull request merged**